### PR TITLE
fix: meta data in package.json was not uploaded

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- add "meta" as a part of the "Package" model

## Why is this pull request needed?
- meta data in a root package's package.json file was not uploaded

## Issues related to this change
none

### CODE REVIEW

File: dm_cli/domain.py [hunk 1]
```diff
@@ -6,13 +6,19 @@
...
+        uid: UUID = None,
+        is_root: bool = False,
+        meta: dict = None,
     ):
         self.name = name
         self.description = description
         self.uid = uid if uid else uuid4()
         self.is_root = is_root
         self.content: List[Union[Package, dict]] = []
+        self.meta: dict | None = meta if meta else {}
```
Comment (netr0m) [inline dm_cli/domain.py:21]:
```suggestion
        self.meta: Union[dict, None] = meta if meta else {}
```
in case we want compatibility with py3.8?

**Review by eoaksnes — APPROVED** [review]
Nice

**Review by netr0m — APPROVED** [review]
LGTM!

File: dm_cli/domain.py (hunk 1 above)
Comment (soofstad) [reply dm_cli/domain.py:21]: fixed

**Pull request merged**